platform/values.yaml

```diff
@@ -1,85 +1,132 @@
+  cluster:
+    enabled: false
+openldap:
+  fullnameOverride: ldap-openldap
+  enabled: true
+  adminPassword: ''
+  env:
+    LDAP_BASE_DN: dc=example,dc=org
+  customLdifFiles:
+    init.ldif : |
+      dn: ou=orgs,dc=example,dc=org
+      objectClass: top
+      objectClass: organizationalUnit
+      ou: orgs
 andi:
   enabled: true
+  image:
+    tag: 0.6.0
   redis:
-    host: platform-redis-master
+    host: redis
   strimzi:
     kafka:
       brokers: streaming-service-kafka-bootstrap:9092
+  service:
+    type: ClusterIP
 discovery-api:
-  enabled: true
-  replicaCount: 1
-  redis:
-    host: platform-redis-master
-  ldap:
-    host: ldap
-discovery-streams:
-  enabled: false
-discovery-ui:
-  enabled: true
-flair:
-  enabled: true
-  redis:
-    host: platform-redis-master
-  strimzi:
-    kafka:
-      brokers: streaming-service-kafka-bootstrap:9092
-forklift:
-  enabled: true
+  image:
+    tag: 0.16.2
   redis:
-    host: platform-redis-master
+    host: redis
+  presto:
+    url: http://kdp-kubernetes-data-platform-presto:8080
+  postgres:
+    host: scosdev.postgres.database.azure.com
   strimzi:
     kafka:
       brokers: streaming-service-kafka-bootstrap:9092
-kdp:
+  s3:
+    hostedFileBucket: hosted-bucket
+  ldap:
+    host: ldap-openldap
+    base: "dc=example,dc=org"
+  vault:
+    endpoint: ""
+  environment: demo
+  service:
+    type: LoadBalancer
+discovery-streams:
+  enabled: false
+  image:
+    environment: demo
+    tag: 0.12.4
+  service:
+    type: LoadBalancer
+kubernetes-data-platform:
+  fullnameOverride: kdp
   enabled: false
-odo:
+  global:
+    environment: demo
+    objectStore:
+      bucketName: ''
+      accessKey: ''
+      accessSecret: ''
+    ingress:
+      annotations: {}
+  presto:
+    workers: 2
+    ingress:
+      enable: false
+      annotations: {}
+    jvm:
+      maxHeapSize: 1536M
+    deploy:
+      container:
+        resources:
+          limits:
+            memory: 2Gi
+            cpu: 2
+          requests:
+            memory: 2Gi
+            cpu: 1
+  minio:
+    enable: true
+    azure:
+      enable: false
+  postgres:
+    enable: false
+    service:
+      externalAddress: ''
+    db:
+      name: metastore
+      user: ''
+      password: ''
+  metastore:
+    allowDropTable: true
+    timeout: 360m
+forklift:
+  fullnameOverride: forklift
   enabled: true
+  image:
+    tag: 0.9.0
+  kafka:
+    brokers: streaming-service-kafka-bootstrap:9092
+  kdp:
+    url: "http://kdp-kubernetes-data-platform-presto:8080"
   redis:
-    host: platform-redis-master
+    host: redis
+  prometheus:
+    scrape: false
 reaper:
+  fullnameOverride: reaper
   enabled: true
+  image:
+    tag: 0.10.1
   redis:
-    host: platform-redis-master
+    host: redis
   strimzi:
     kafka:
       brokers: streaming-service-kafka-bootstrap:9092
   vault:
     endpoint: ""
-redis:
-  enabled: true
-  usePassword: false
-  cluster:
-    enabled: false
 valkyrie:
+  fullnameOverride: valkyrie
```

Comment:
This seems... redundant.

Reply:
If you don't do this some of the pods get prepended with platform.  Such as ```platform-valkyrie-as912901012ad2```

Reply:
Ah, gotcha. 👍